src/views/TableView/TableView.js

```diff
@@ -0,0 +1,31 @@
+import PropTypes from "prop-types";
+import React, { Component } from "react";
+import { withRouter } from "react-router-dom";
+import { compose } from "recompose";
+import Table from "../../components/Table";
+import { withTableData } from "../../data/withTableData";
+import { tableShape } from "../../types";
+
+class TableView extends Component {
```

Comment:
Why not a functional component here?

Reply:
You're right, I missed this one, I was moving code around to accommodate for the routing and I overlooked it.